repentogon/LuaInterfaces/CustomCallbacks.cpp

```diff
@@ -4313,21 +4313,21 @@ HOOK_METHOD(ItemPool, ForceAddPillEffect, (int32_t ID)->int) {
 	};
 
 //POST_FORCE_PILL_EFFECT (1129)
-	callbackid = 1129;
-	if (CallbackState.test(callbackid - 1000)) {
-		lua_State* L = g_LuaEngine->_state;
-		lua::LuaStackProtector protector(L);
+callbackid = 1129;
+if (CallbackState.test(callbackid - 1000)) {
+	lua_State* L = g_LuaEngine->_state;
+	lua::LuaStackProtector protector(L);
 
-		lua_rawgeti(L, LUA_REGISTRYINDEX, g_LuaEngine->runCallbackRegistry->key);
+	lua_rawgeti(L, LUA_REGISTRYINDEX, g_LuaEngine->runCallbackRegistry->key);
 
-		lua::LuaResults result = lua::LuaCaller(L).push(callbackid)
-			.pushnil()
-			.push(ID)
-			.push(ret)
-			.call(1);
-	};
+	lua::LuaResults result = lua::LuaCaller(L).push(callbackid)
+		.pushnil()
+		.push(ID)
+		.push(ret)
+		.call(1);
+};
 
-	return ret;
+return ret;
```

Comment:
What's with the diffs up here? Accidental?

Reply:
oh. i think visual studio's formatter messed something up. i will fix it.